go.mod

```diff
@@ -2,91 +2,88 @@ module github.com/lightninglabs/lndinit
 
 require (
 	github.com/btcsuite/btcd v0.23.5-0.20230228185050-38331963bddd
-	github.com/btcsuite/btcwallet v0.16.10-0.20230517173256-aa62c04afcdf
-	github.com/jessevdk/go-flags v1.4.0
+	github.com/btcsuite/btcwallet v0.16.10-0.20230621165747-9c21f464ce13
+	github.com/jessevdk/go-flags v1.5.0
 	github.com/kkdai/bstream v1.0.0
 	github.com/lightninglabs/protobuf-hex-display v1.4.3-hex-display
-	github.com/lightningnetwork/lnd v0.16.3-beta
-	github.com/stretchr/testify v1.8.1
-	google.golang.org/grpc v1.41.0
```

Comment:
Are we able to restrict the other updates here? How'd you bump the version locally? 

Mentioning as there's some gRPC/LNC related stuff. we've fixed in some later versions re L402 compat, wanting to make sure we don't indirectly break things.

Reply:
Usually I do:
  * edit this manually to be `github.com/lightningnetwork/lnd v0.16.4-beta.rc1`
  * run `go mod tidy`

Reply:
Cool, looks much better in the latest diff.